system/jlib/jptree.hpp

```diff
@@ -24,6 +24,9 @@
 #include "jiter.hpp"
 #include "jprop.hpp"
 
+#include <list>
```

Comment:
Should now be 
#include <initializer_list>  

Reply:
yes, will change.